hr_payslip_ytd_amount/i18n/fr.po

```diff
@@ -1,32 +1,104 @@
 # Translation of OpenERP Server.
 # This file contains the translation of the following modules:
-#	* hr_payslip_ytd_amount
+# 	* hr_payslip_ytd_amount
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: OpenERP Server 7.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2014-12-22 01:03+0000\n"
-"PO-Revision-Date: 2014-12-22 01:03+0000\n"
-"Last-Translator: <>\n"
+"POT-Creation-Date: 2015-02-12 18:29+0000\n"
+"PO-Revision-Date: 2015-02-12 13:34-0500\n"
+"Last-Translator: David Dufresne <david.dufresne@savoirfairelinux.com>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: \n"
+"Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
 
 #. module: hr_payslip_ytd_amount
-#: field:hr.payslip.line,total_ytd:0
-msgid "Total Year-to-date"
+#: report:payslip_ytd_amount:0
+msgid "Note"
+msgstr "Note"
+
+#. module: hr_payslip_ytd_amount
+#: report:payslip_ytd_amount:0
+msgid "Reference"
+msgstr "Référence"
+
+#. module: hr_payslip_ytd_amount
+#: report:payslip_ytd_amount:0
+#, fuzzy
+msgid "Amount Year-to-date"
```

Comment:
Translation is fuzzy it might not get translated and instead of "Total Année Courante" "Total de l'année courante"